src/commands/serve.rs

```diff
@@ -76,6 +78,31 @@ pub(crate) fn serve(args: &ArgMatches) {
 }
 
 fn serve_foreground(args: &ArgMatches) {
+    let path = get_rymfony_project_directory().unwrap();
+    let rymfony_pid_file = path.join("rymfony.pid");
+    debug!("Search PID FILE {}", rymfony_pid_file.to_str().unwrap());
+    if rymfony_pid_file.exists() {
+        //Check if process is rymfony and exit if true.
+        debug!("Information file {} found", rymfony_pid_file.to_str().unwrap());
+
+        let infos: ProcessInfo = serde_json::from_str(read_to_string(&rymfony_pid_file).unwrap().as_str()).expect("Unable to unserialize data");
+
+        let mut system = sysinfo::System::new_all();
+        system.refresh_all();
+        let mut found = false;
+        for (pid, proc_) in system.get_processes() {
+            if pid == &infos.pid() && proc_.exe().to_str().unwrap().ends_with("rymfony") {
+                found = true;
```

Comment:
A `break` statement could be useful here.

Reply:
`break` added